SentinelOne/sentinelone/ingest/parser.yml

```
@@ -541,7 +541,14 @@ stages:
           sentinelone.data.uploadedFilename: "{{json_event.message.data.uploadedFilename}}"
           sentinelone.data.uuid: "{{json_event.message.data.uuid}}"
 
-          user.name: "{{json_event.message.data.userName or json_event.message.data.username or json_event.message.data.lastLoggedInUserName}}"
+          user.name: "{{json_event.message.data.lastLoggedInUserName}}"
```

I think that `user.name` must remain:
```suggestion
          user.name: "{{json_event.message.data.userName or json_event.message.data.username or json_event.message.data.lastLoggedInUserName}}"
```